cmd/validator/validator.go

```diff
@@ -17,6 +17,10 @@ optional flags:
     	Subdirectories to exclude when searching for configuration files
   -exclude-file-types string
     	A comma separated list of file types to ignore
+  -file-types string
+    	A comma separated list of file types to validate. Mutually exclusive with -exclude-file-types
+  -output
+     	Destination of a file to outputting results
```

Comment:
Output is no longer a flag since the new multi-reporter logic has displaced it. Please remove
```suggestion
```

Reply:
Done.